core-primitives/enclave-api/src/teeracle_api.rs

```diff
@@ -29,6 +29,9 @@ pub trait TeeracleApi: Send + Sync + 'static {
 		crypto_currency: &str,
 		fiat_currency: &str,
 	) -> EnclaveResult<Vec<u8>>;
+
+	/// update weather data for the corresponding coordinates.
```

Comment:
We are kind of strict with our docs. We want it to be full sentences and hence start with a capital letter. :)